### DISCUSSION TITLE
How much preprocessing is required to get acceptable results?

### DISCUSSION THREAD

**Discussion by Naeemkh** [NSAPH-Software/CausalGPS#153] [Ideas]
How much preprocessing is required to get acceptable results?

This topic includes discussions on the amount of preprocessing on the input data.

- Should we impute missing values?
- Should we remove potential instrumental variables or add a function to double-check it (based on correlation with the outcome) and raise a warning if some of the features can be considered instrumental variables and require further investigation. 
- Can we just provide all available features and hope for the best, or do we need to select a subset of features.

**Comment by daniellebraun** [comment]
These are good suggestions but I think they should all be on the user side. We can make recommendations for them to check these, but I think preprocessing should be on users.
